<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PRC-20170801ZCL" name="automate.webautomate.testcase.Login" tests="7" failures="0" timestamp="18 Sep 2019 03:43:28 GMT" time="39.911" errors="1">
  <testcase name="FalseULogin" time="8.504" classname="automate.webautomate.testcase.Login"/>
  <testcase name="logingTrue" time="15.448" classname="automate.webautomate.testcase.Login"/>
  <testcase name="FalsePNullLogin" time="2.732" classname="automate.webautomate.testcase.Login"/>
  <testcase name="FalsePLogin" time="2.839" classname="automate.webautomate.testcase.Login"/>
  <testcase name="LoginFindPasswordPage" time="5.187" classname="automate.webautomate.testcase.Login">
    <error type="java.lang.AssertionError" message="找回密码 expected:&lt;找回密码&gt; but was:&lt;产品登录页面：点击【忘记密码】跳转找回密码页面,断言title&gt;">
      <![CDATA[java.lang.AssertionError: 找回密码 expected:<找回密码> but was:<产品登录页面：点击【忘记密码】跳转找回密码页面,断言title>
	at org.testng.AssertJUnit.assertEquals(AssertJUnit.java:101)
	at automate.webautomate.testcase.Login.LoginFindPasswordPage(Login.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:330)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </error>
  </testcase> <!-- LoginFindPasswordPage -->
  <testcase name="JUPclick" time="2.476" classname="automate.webautomate.testcase.Login"/>
  <testcase name="FalseUNullLogin" time="2.725" classname="automate.webautomate.testcase.Login"/>
</testsuite> <!-- automate.webautomate.testcase.Login -->
